src/redisearch_rs/inverted_index/src/types/mod.rs

```diff
@@ -7,7 +7,27 @@
  * GNU Affero General Public License v3 (AGPLv3).
 */
 
+use ffi::RSIndexResult;
+use std::ffi::c_int;
+
 /// Represents a numeric value in an index record.
 /// cbindgen:field-names=[value]
 #[repr(C)]
 pub struct RSNumericRecord(pub f64);
+
+/// Represents an aggregate array of values in an index record.
+/// cbindgen:rename-all=CamelCase
+#[repr(C)]
+pub struct RSAggregateResult {
+    /// The number of child records
+    pub num_children: c_int,
+
+    /// The capacity of the records array. Has no use for extensions
+    pub children_cap: c_int,
+
+    /// An array of records
+    pub children: *mut *mut RSIndexResult,
+
+    /// A map of the aggregate type of the underlying records
+    pub type_mask: u32,
```

Comment:
This is not "just" a `u32`.
It's a bitflags, and we should model it as such. `bindgen` should then be able to translate it to a "plain" u32 over the FFI boundary.